setup.py

```diff
@@ -27,6 +27,7 @@
         'tables<=3.4.0',
         'pandas<0.25',
 
+        'dismod_mr',
```

Comment:
consider a pin here, I suppose

Reply:
Don't put dismod in the main requirements.  Put it in an extras block.  It brings in old pymc which has nasty requirements that might conflict with vivarium stuff.